src/handler.ts

```diff
@@ -15,9 +15,9 @@ export const handle = async (evt: SQSEvent): Promise<BatchRes[]> => {
   const rs = toReqs(evt.Records)
   try {
     return await sendHooks(rs)
-  } catch (err) {
+  } catch (err: any) {
     error(name, err)
-    if (err.message !== BATCH_ERROR) return await sendErrorBatch(rs)
-    else throw err // Batch won't be deleted from queue and will be retried
+    if (err.message !== BATCH_ERROR) return sendErrorBatch(rs)
+    else return Promise.reject(err) // Batch won't be deleted from queue and will be retried
```

Comment:
do we have any tests to cover this change? This is a change that feels a little more functional vs syntactical

Reply:
There is an existing test case for this in the [handler test file](https://github.com/Dwolla/webhook-handler/blob/cf571dc1b7d9c380715dfdcab0781145d5635429/test/handler.test.ts#L33-L44)